packages/utilities/src/styled.tsx

```diff
@@ -26,13 +26,13 @@ const DefaultFields = ['theme', 'styles'];
  * The styled HOC wrapper allows you to create a functional wrapper around a given component which will resolve
  * getStyles functional props, and mix customized props passed in using concatStyleSets.
  *
- * @example
- * ```tsx
+ * Example:
```

Comment:
This is also regressive, but isn't as serious as the `@defaultvalue`. Would be good to understand where support for `@example` is on API Extractor's roadmap.

Reply:
@cliffkoh The `@example` tag is already part of TSDoc and will be accepted in API Extractor 6.  The syntax is discussed in https://github.com/Microsoft/tsdoc/issues/20

If you have suggestions/questions, please add them to that issue.